src/stores/viewStore.ts

```diff
@@ -43,7 +43,7 @@ interface ViewStoreActions {
     filterLogs: (filter: LogLevelFilter) => void;
 
     loadPageByAction: (navAction: NavigationAction) => void;
-    updateIsPrettified: (newIsPrettified: boolean) => void;
+    setIsPrettified: (newIsPrettified: boolean) => void;
```

Comment:
(i understand this is a WIP PR and the changes are not finalized yet)

for all Zustand store actions that do not simply change the value of a state, we prefer naming them with a `updatePrefix`. @coderabbitai create an issue for adding this in the dev docs